integration_tests/test_version.py

```diff
@@ -0,0 +1,7 @@
+"""Test this cubedash build has a valid version
+"""
+import cubedash
+
+
+def test_check_version():
+    assert cubedash.__version__ != "Not-Installed"
```

Comment:
This will behave strangely with Docker in the dev environment, I think, because the code that is being run has _not_ been installed. I'm not sure this test is a good idea.